src/Utils/cameraPermissionHandler.ts

```
@@ -0,0 +1,30 @@
+import { useCallback, useState } from "react";
+import { toast } from "sonner";
+
+export const handleCameraPermission = () => {
+  const [toastShown, setToastShown] = useState(false);
+
+  const requestPermission = useCallback(
+    async (device: "camera", cameraFacingMode: string = "user") => {
+      try {
+        setToastShown(false); 
```

ESLint

[error] Delete `·`

(prettier/prettier)